subprojects/internal-integ-testing/src/main/groovy/org/gradle/test/fixtures/keystore/TestKeyStore.groovy

```diff
@@ -32,12 +32,24 @@ class TestKeyStore {
         new TestKeyStore(rootDir)
     }
 
+    /*
+     clientStore/serverStore only contains self-signed certificates for embedded HTTPS server.
+     To make the client work with both embedded HTTPS server and real-world HTTPS server (e.g. Maven Central),
+     we need to merge JDK's cacerts into the custom keystore/truststore via:
+
+     keytool -importkeystore -srckeystore <JDK cacerts file location> -destkeystore <resource>/test-key-store/keyStore
+     keytool -importkeystore -srckeystore <JDK cacerts file location> -destkeystore <resource>/test-key-store/trustStore
+
+     Note that default password for JDK cacerts is "changeit".
+
+     The current keyStore-adoptopenjdk-8/trustStore-adoptopenjdk-8 is created from AdoptOpenJDK8 cacerts.
+     */
     private TestKeyStore(TestFile rootDir) {
         keyStore = rootDir.file("clientStore")
         trustStore = rootDir.file("serverStore")
 
-        copyCertFile("test-key-store/keyStore", keyStore)
-        copyCertFile("test-key-store/trustStore", trustStore)
+        copyCertFile("test-key-store/keyStore-adoptopenjdk-8", keyStore)
```

Comment:
I think we only should add the keys to the trust store, I don't see why we need any keys in the key store.

Reply:
Let's remove it and see if anything breaks.